src/main/java/com/google/alpollo/ConfigHelper.java

```diff
@@ -35,11 +34,16 @@ private ConfigHelper() {
    * or it was incorrect/didn't have projectID field
    */
   public static String getProjectID() {
-    try (InputStream inputStream = ClassLoader.getSystemResourceAsStream(CONFIG_FILE_NAME);
+    try (InputStream inputStream = new FileInputStream(new File(CONFIG_FILE_PATH));
          final Reader reader = new InputStreamReader(Objects.requireNonNull(inputStream))) {
       return gson.fromJson(reader, ConfigInfo.class).getProjectID();
     } catch (Exception parseException) {
-      return null;
+      try (InputStream inputStream = ClassLoader.getSystemResourceAsStream(CONFIG_FILE_NAME);
+           final Reader reader = new InputStreamReader(Objects.requireNonNull(inputStream))) {
```

Comment:
The same code as before. And which kind of exceptions you catch?

Reply:
yes, I don't know how don't repeat it and make it good normal

Reply:
I catch any exception actually, more specific nullpointer and json exceptions